supplier_app/tests/test_view.py

```diff
@@ -563,7 +563,13 @@ def setUp(self):
             cuit='20392379685',
             company=self.company2,
         )
-
+        self.taxpayer_usa1 = TaxPayerUnitedStatedFactory(
+            workday_id='3',
+            taxpayer_state=STATUS_PENDING,
+            taxpayer_id_number='12345678',
+            company=self.company2,
+            country='USA',
+        )
```

Comment:
are you using it?

Reply:
Yes, It is save for searching data for the table

Reply:
If you delete it, is any going to fail?